docs/source/contents/4_draft_mapping.rst

```diff
@@ -1662,6 +1662,407 @@ opaque:hostItem
 location
 ========
 
++-----------------------------------+----------------+-------------------+-------------------------------------------------------------------------+
+| Predicate                         | Value Type     | Range (if needed) | Usage Notes                                                             |
++===================================+================+===================+=========================================================================+
+| relators:rps                      | Literal or URI |                   | Use for <mods:physicalLocation> values, preferably using                |
+|                                   |                |                   | a URI for the organization from a controlled vocabulary                 |
+|                                   |                |                   | such as VIAF of Library of Congress Real World Objects.                 |
++-----------------------------------+----------------+-------------------+-------------------------------------------------------------------------+
+| skos:note                         | Literal        |                   | Use to note <mods:shelfLocator> strings.                                |
++-----------------------------------+----------------+-------------------+-------------------------------------------------------------------------+
+| dbo:collection                    | Literal        |                   | Use to note <mods:physicalLocation[@displayLabel=Collection]> strings.  |
```

Comment:
The way you have your XML is weird. It's not well formed xml or an xpath.

Switch to 

```
:code:`mods:physicalLocation[@displayLabel="Collection"]`
```

 or  

```
:code:`<mods:physicalLocation displayLabel="Collection">`
```
I don't care so much about the inline code bit, but pick xml or xpath.

Reply:
I think we may have more of the Xpath already present in the mapping doc, so I'd say that would be more consistent.